schema-nft-marketplace.graphql

```diff
@@ -232,11 +232,11 @@ type CollectionDailySnapshot @entity {
     " Royalty fee rate in percentage. E.g. 2.5% should be 2.5 "
     royaltyFee: BigDecimal!
 
-    " Number of tokens listed for sale in the marketplace. "
-    listedCount: BigInt!
+    " Minimum sale price of the day (in ETH) "
+    minSalePrice24hETH: BigDecimal!
```

Comment:
why not `dailyMinSalePriceETH`? It'll be consistent with other schemas. Also 24h sounds like it's a rolling window